lib/providers/map_data.dart

```diff
@@ -98,8 +78,11 @@ class MapData extends ChangeNotifier {
     notifyListeners();
   }
 
-  final controllerStarting = TextEditingController();
-  final controllerDestination = TextEditingController();
+  void removeItinerary() {
```

Comment:
can you specify what this function is doing? Is it responsible of hiding the directions panel? if yes, please specify it in a comment or make the function name more informative.

Reply:
Check 
4b36668